cmd/create.go

```diff
@@ -4,16 +4,16 @@ Copyright © 2023 NAME HERE <EMAIL ADDRESS>
 package cmd
 
 import (
-	"context"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/signal"
 	"syscall"
 
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v3"
+	"gopkg.in/yaml.v2"
```

Comment:
What's the reason to switch back to v2 of the yaml package?

Reply:
This happened by accident somehow, I'll revert this change.